hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -309,6 +309,18 @@ public void rollbackFailedBootstrap() {
    */
   public abstract O insert(I records, final String instantTime);
 
+  /**
+   * Inserts the given Error HoodieRecords, into the table. This API is intended to be used for normal writes.
+   * <p>
+   * This API is intended to be used to write wrong records.
+   *
+   * @param records Error HoodieRecords to insert
+   * @param instantTime Instant time of the commit
+   * @return Collection of WriteStatus to inspect errors and counts
+   */
```

Comment:
also can we add ad API such as getErrorTableWriter()  . Such as in HoodieSparkSqlWriter.write() when error occurs before clietn.insert/upsert can also write the errors record. cc @xushiyan